src/Features/CSharp/Portable/MakeMethodAsynchronous/CSharpMakeMethodAsynchronousCodeFixProvider.cs

```diff
@@ -90,6 +94,10 @@ protected override bool IsAsyncSupportingFunctionSyntax(SyntaxNode node)
                     // in Task<>.
                     newReturnType = taskOfTType.Construct(methodSymbol.ReturnType).GenerateTypeSyntax();
                 }
+                else
+                {
+                    newReturnType = newReturnType.WithAdditionalAnnotations(Formatter.Annotation);
```

Comment:
@sharwell I like this solution, but in most cases it is useless operation, but if you ok with that I will update PR